src/Illuminate/Support/helpers.php

```diff
@@ -328,6 +328,7 @@ function retry($times, callable $callback, $sleepMilliseconds = 0, $when = null)
         $attempts++;
         $times--;
 
+        /** @phpstan-ignore return.missing */
```

Comment:
I think in this case it is a false positive of PHPStan. This method throws an exception if the retry limit is exceeded, so it never returns `null`.